components/brandprofile/ColorShowcase.tsx

```
@@ -112,23 +171,242 @@ const ColorSwatch = ({ color, className = '' }: ColorSwatchProps) => {
+            <span className="relative z-10">Sample Text</span>
+            {copiedValue && (
+              <div className="absolute inset-0 bg-black bg-opacity-50 flex items-center justify-center">
+                <span className="text-white text-xs">Copied!</span>
+              </div>
+            )}
+          </div>
+          <div className="space-y-1">
+            <button
+              onClick={() => handleCopy(combination.lightBackground)}
+              className="text-xs text-secondary-600 dark:text-secondary-400 hover:text-primary-600 dark:hover:text-primary-400 transition-colors block"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)